app/Controllers/importExportController.php

```diff
@@ -50,12 +50,21 @@ private static function megabytes(string $size_str) {
 	 */
 	private static function minimumMemory($mb): void {
 		$mb = (int)$mb;
+		if (ini_get('memory_limit') === false) {
+			return;
+		}
 		$ini = self::megabytes(ini_get('memory_limit'));
```

Comment:
We still want to set the memory limit even when it was not defined / default. Also avoid redundant system / expensive calls:

```suggestion
		$ini = self::megabytes(ini_get('memory_limit') ?: '0');
```